SQL/Archive/17.1/2017-07-28_survey_accounts_password_storge.sql

```diff
@@ -0,0 +1,3 @@
+-- The OneTimePassword storage capacity should be extended according to new key generation logic
+
+ALTER TABLE participant_accounts MODIFY COLUMN OneTimePassword VARCHAR(255) ;
```

Comment:
this works well but my only recommendation would be to try to see if you can use VARCHAR(16) instead in order to save some space, but maybe that is a little too nitpicky...